_data/events_gdcr2018/Hyderabad-India-CDKGlobal.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Coderetreat @ CDK Global (India) Pvt. Ltd.",
+  "url": "https://sites.google.com/view/cdkhyderabadevents",
+  "moderators": [
+    "Vibhor.Mamgain@cdk.com",
```

Comment:
[Same remark](https://github.com/coderetreat/coderetreat.github.io/pull/366#discussion_r224328090) as in #366 - would be nice to either use names (nicknames or abbreviated names are fine) or twitter handles. We'll be reaching out to you on Twitter and will promote your event with the moderators mentioned.